deploy/charts/sdp-prototype/values.yaml

```diff
@@ -21,14 +21,14 @@ workflow:
     version: 0.0.7
     replicas: 2
     resources: {}
-    # Allow direct Kubernetes deployments? Should be deprecated.
+    # Allow direct Kubernetes deployments? Should be deprecated
     direct-kube-deploy: false
   testdask:
     container: pw410/workflow-testdask
     version: 0.0.1
     replicas: 2
     resources: {}
-    imagePullPolicy: IfNotPresent
+    imagePullPolicy: Never
```

Comment:
I don't understand the image pull policy setting here.

Reply:
Yeah, that's a mistake. For the record: an image pull policy of "Never" is a good idea when you're developing locally, as you do not want Kubernetes to attempt to synchronise with Docker Hub.